packages/thirdweb/src/extensions/farcaster/read/getRegistrationPrice.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, it, expect, vi, afterEach } from "vitest";
+
+import { getRegistrationPrice } from "./getRegistrationPrice.js";
+import { TEST_CLIENT } from "~test/test-clients.js";
+
+const fetchSpy = vi.spyOn(globalThis, "fetch");
+
+describe("farcaster.getRegistrationPrice", () => {
+  afterEach(() => {
+    fetchSpy.mockClear();
+  });
+
+  it("should return the price to register a new fid", async () => {
+    const price = await getRegistrationPrice({
+      client: TEST_CLIENT,
+    });
+    expect(price).toBe(814540083517510n);
```

Comment:
these tests actually hit network right? that value will prob change over time.

Prob fine for right now, but we should make this hit an anvil fork of OP instead, at a fixed block.

might need a bit of setup but worth it, we're gonna need a multichain test setup soon. cc @jnsdls

Reply:
actually not fine even right now :p already getting a different value

Reply:
Oh I thought the tests targeted a frozen block, will fix

Reply:
I added second forked chain that runs alongside the mainnet one. Not the most scalable solution if more chains are needed later but should do for now

Reply:
I think that's actually fine, we'll likely end up with:
- mainnet fork
- OP stack fork
- <insert stack here> fork

We'll likely never need to fork *all* chains but we likely will want at least one per type of chain